python/llm/README.md

```diff
@@ -87,14 +91,25 @@ You may run the models using `transformers`-style API in `bigdl-llm`.
 
   See the complete examples [here](example/transformers/transformers_int4/).  
 
-  >**Note**: You may apply more low bit optimizations (including INT8, INT5 and INT4) as follows: 
-  >```python
-  >model = AutoModelForCausalLM.from_pretrained('/path/to/model/', load_in_low_bit="sym_int5")
-  >```
-  >See the complete example [here](example/transformers/transformers_low_bit/).
-
+  >**Notes**: 
+  
+  * You may apply more low bit optimizations (including INT8, INT5 and INT4) as follows: 
+    >```python
+    >model = AutoModelForCausalLM.from_pretrained('/path/to/model/', load_in_low_bit="sym_int5")
+    >```
+    >See the complete example [here](example/transformers/transformers_low_bit/).
   
-  After the model is optimizaed using INT4 (or INT5/INT8), you may save and load the optimized model as follows:
+  *  You may also load the Hugging Face model with `transformers`-style API developed for specific models (`Llama`, `Gptneox`, `Bloom`, `Starcoder`, `ChatGLM`) as below:
+
+
+      >```python
+      >#specify native to `False` to load Hugging Face Transformers Llama model with INT4 optimizations
+      >from bigdl.llm.transformers import LlamaForCausalLM
+      >model = LlamaForCausalLM.from_pretrained('/path/to/model/', native=False, load_in_4bit=True)
+      >```
+
+  After the model is optimizaed using INT4 (or INT8/INT5), you may save and load the optimized model as follows:
```

Comment:
I don't think this part needs to be added in the main README.
For non-native models, we will always recommend for AutoModel